dot/parachain/statement-distribution/cluster_tracker_test.go

```diff
@@ -210,3 +210,156 @@ func TestClusterTracker_receive_statements(t *testing.T) {
 		)
 	})
 }
+
+// tests that cover clusterTracker.canSend() and clusterTracker.noteSent()
+func TestClusterTracker_send_statements(t *testing.T) {
+	group := []parachaintypes.ValidatorIndex{5, 200, 24, 146}
+	secondingLimit := uint(2)
+	hashA := parachaintypes.CandidateHash{Value: common.Hash{0x1}}
+	hashB := parachaintypes.CandidateHash{Value: common.Hash{0x2}}
+	hashC := parachaintypes.CandidateHash{Value: common.Hash{0x3}}
+
+	t.Run("accepts_incoming_valid_after_outgoing_seconded", func(t *testing.T) {
+		tracker := newClusterTracker(group, secondingLimit)
```

Comment:
add `t.Parallel()` to all the test cases
```suggestion
	t.Parallel()
		tracker := newClusterTracker(group, secondingLimit)
```

Reply:
I've [added that on the other branch](https://github.com/ChainSafe/gossamer/pull/4773/commits/82b5e741137d38dfbb6bdd11d119dc9b44001916) so should be fine once both are merged.